MeatcubeParkingSystem/src/main/java/com/metacube/parking/dao/ConnectionClass.java

```diff
@@ -0,0 +1,45 @@
+package com.metacube.parking.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+/**
+ * The Class ConnectionClass for setting up the jdbc connection .
+ * @author Aman Gautam
+ * 
+ * Dated :21/9/19
+ */
+public class ConnectionClass {
+
+	public static Connection connect;
+	/** 
+	 * @return object of Connection class
+	 * @throws Exception when driver class not found or can't create connection
+	 */
+	public Connection getConnection() throws Exception {
+		try 
+		{
+			Class.forName("com.mysql.jdbc.Driver");
+			connect = (Connection) DriverManager.getConnection("jdbc:mysql://localhost:3306/employee", "root", "1083543569");
+			System.out.println("Remote DB connection established");
+		} catch (ClassNotFoundException e) 
```

Comment:
Can handle multiple exception types which performs the same functionality in single catch by using `OR` operator